pkg/karmadactl/join.go

```diff
@@ -106,12 +105,19 @@ func (j *CommandJoinOption) Complete(args []string) error {
 }
 
 // Validate checks option and return a slice of found errs.
-func (j *CommandJoinOption) Validate() error {
+func (j *CommandJoinOption) Validate(args []string) error {
+	var errs []error
+	if len(args) > 1 {
+		errs = append(errs, fmt.Errorf("only the cluster name is allowed as an argument"))
+	}
+	if len(j.ClusterName) == 0 {
+		errs = append(errs, fmt.Errorf("cluster name is required"))
```

Comment:
I think directly throwing the error seems better. What are advantages using `errs`?

Reply:
In my opinion，If the user has multiple errors, all errors can be returned to the user through the errs. Otherwise, it may take several times for the user to correct the command

Reply:
Makes sense, but the errors are progressive, it seems rare that they are happending at the same time

Reply:
Should I change it to return whenever there is an error?

Reply:
> Should I change it to return whenever there is an error?

I think it is ok, because we also directly return an error in other `validate` function.

Reply:
thx, has been modified